### PR TITLE
fix(plugins): if gate doesn't have plugin-manifest.json, ignore the error

### PR DESCRIPTION
**Pull request by bpowell**

### CODE REVIEW

File: app/scripts/modules/core/src/plugins/plugin.registry.ts [hunk 1]
```diff
@@ -115,6 +115,10 @@ export class PluginRegistry {
       .get()
       .catch((error: any) => {
         console.error(`Failed to load ${uri} from ${source}`);
+        // If we cannot hit the Gate URL, ignore it
+        if (error.data.status === 404) {
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/plugins/plugin.registry.ts]: How could this happen?  This endpoint should always be in gate, as far as I can tell

Comment (bpowell) [reply app/scripts/modules/core/src/plugins/plugin.registry.ts]: Not if you are playing with master of Deck and Gate that hasn't been updated in a while. Both @claymccoy and I found this while doing some testing

Comment (christopherthielen) [reply app/scripts/modules/core/src/plugins/plugin.registry.ts]: ok, but let's delete this in a month or two.

**Comment by christopherthielen** [comment]
@bpowell if you mark your pull requests as "allow edits from maintainers" then mergify will keep your PRs up to date with master automatically.

**Pull request merged**